barna-ajedrez/server/models/TournamentModel.js

```diff
@@ -0,0 +1,112 @@
+const CONFIG = require('../config');
+const Sequelize = require('sequelize');
+const sequelize = new Sequelize('ajedrezdb', CONFIG.duser, CONFIG.dpass, {
```

Comment:
You're connecting once per model. Connect once and share the connection between models.